documents/Libraries/stdlib/osl/stdosl.h

```diff
@@ -0,0 +1,601 @@
+/////////////////////////////////////////////////////////////////////////////
```

Comment:
Adding in the stdosl header, as it's needed when running our OSL tests. Hence our MATERIALX_OSL_INCLUDE_PATH can now point to this directory (documents/Libraries/stdlib/osl) to make sure the same OSL headers are always used when we running our tests.